README.md

```diff
@@ -81,6 +95,7 @@ These are the options that you can set (check the [API documentation for `Puck::
 * `:build_dir`: defaults to `build`, but if you want the Jar file to end up somewhere else you can change it with this option.
 * `:jruby_complete`: if you don't want to depend on the `jruby-jars` gem for some reason you can provide the path to your own `jruby-complete.jar`
 * `:extra_files`: a list of files to include in the Jar. The option can be either an Array, in which case paths must be below the `:app_dir`, or a Hash, in which case the file specified by the key is included at the path specified by the corresponding value.
+* `:merge_archives`: a list of Jars to be merged into the Jar. The option can be either an Array, then the external Jar will be merged on the root level of the Jar, or a Hash, in which case the Jar specified by the key is merged at the path specified by its value. Signature files ('META-INF/\*.SF', 'META-INF/\*.RSA', 'META-INF/\*.DSA' and 'META-INF/SIG-\*') are discarded in the merge.
```

Comment:
"… either an Array, _in which case_ the _source Jar or zip file_ will be merged _at_ the root"

and

"are discarded in the merge, _since they describe the source Jar and will not match the Jar produced by Puck_."